src/menu.tsx

```diff
@@ -80,6 +80,9 @@ export function get_menu_items(
     const supportsTerminal = cockpit.manifests.system?.tools?.terminal?.capabilities?.includes("path");
 
     if (selected.length === 0) {
+        // HACK: basename('/') is currently ""
+        const current_directory = { ...cwdInfo, name: basename(path) || "/", category: null, to: null };
```

Comment:
That was fixed in https://github.com/cockpit-project/cockpit/commit/3c421eb3cb0c16d12a827bb0f5919ba1c906b756 which ought to be in current c-files main?

Doesn't hurt, and I figure this was developed in parallel, just a cleanup opportunity.

Reply:
Oh, the second patch actually does change it, so rebase error